tempesta_fw/sock.c

```diff
@@ -1087,6 +1084,41 @@ ss_getpeername(struct sock *sk, struct sockaddr *uaddr, int *uaddr_len)
 }
 EXPORT_SYMBOL(ss_getpeername);
 
+static inline void
+__ss_do_send(SsWork *sw)
+{
+	struct sock *sk = sw->sk;
+
+	bh_lock_sock(sk);
+	ss_do_send(sk, &sw->skb_list, sw->flags);
```

Comment:
Usually `xxx` function calls `__xxx` or `do_xxx`, so it has sense to rename the functions.